mmdet/ops/nms/nms_wrapper.py

```diff
@@ -116,3 +116,36 @@ def soft_nms(dets, iou_thr, method='linear', sigma=0.5, min_score=1e-3):
     else:
         return new_dets.numpy().astype(dets.dtype), inds.numpy().astype(
             np.int64)
+
+
+def batched_nms(bboxes, scores, idxs, nms_cfg):
+    """Performs non-maximum suppression in a batched fashion.
+
+    Modified from https://github.com/pytorch/vision/blob
+    /505cd6957711af790211896d32b40291bea1bc21/torchvision/ops/boxes.py#L39.
+    In order to perform NMS independently per class, we add an offset to all
+    the boxes. The offset is dependent only on the class idx, and is large
+    enough so that boxes from different classes do not overlap.
+
+    Arguments:
+        bboxes (torch.Tensor): bboxes in shape (N, 4).
+        scores (torch.Tensor): scores in shape (N, ).
+        idxes (torch.Tensor): each index value correspond to a category, and
```

Comment:
- Rename it to `inds`, which is short for `indices` (plural form of index).
- `inds` does not necessarily correspond to categories.